src/Console/Command/FixCommand.php

```diff
@@ -292,8 +294,8 @@ protected function configure()
      */
     protected function execute(InputInterface $input, OutputInterface $output)
     {
-        $stdErr = ($output instanceof ConsoleOutputInterface) ? $output->getErrorOutput() : null;
-        if ($stdErr && extension_loaded('xdebug')) {
+        $stdErr = ($output instanceof ConsoleOutputInterface) ? $output->getErrorOutput() : $output;
+        if (extension_loaded('xdebug')) {
```

Comment:
scenario:
When `$output` is not `ConsoleOutputInterface`, then `$stdErr` will be `$output`.

When format is JSON (or YML), on `stdout` (so `$output`) we will print the JSON to the console, user may grab it and parse as JSON format.
When we add extra content to it, like warning about xdebug, the output will be impossible to parse as JSON. Then the whole output is corrupted and in fact useless.
Take a look at https://github.com/FriendsOfPHP/PHP-CS-Fixer/issues/826 and it's fix https://github.com/FriendsOfPHP/PHP-CS-Fixer/pull/837